src/test/java/org/jabref/logic/importer/fetcher/OpenAccessDoiTest.java

```diff
@@ -28,7 +28,7 @@ void setUp() {
     @Test
     void findByDOI() throws IOException {
         entry.setField(StandardField.DOI, "10.1038/nature12373");
-        assertEquals(Optional.of(new URL("https://dash.harvard.edu/bitstream/1/12285462/1/Nanometer-Scale%20Thermometry.pdf")), finder.findFullText(entry));
+        assertEquals(Optional.of(new URL("https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4221854")), finder.findFullText(entry));
```

Comment:
This looks wrong, it should return the fulltext pdf

Reply:
We should revert this then and get the other changes in 😅